blackbox/docs/admin/monitoring.rst

```diff
@@ -193,6 +193,80 @@ name and has the following attributes:
 |                      | transport protocol                                      |
 +----------------------+---------------------------------------------------------+
 
+ThreadPools MXBean
+------------------
+
+The ``ThreadPools`` MXBean exposes statistic information about the used thread
+pools of a ``CrateDB`` node.
+
+It can be accessed using the ``io.crate.monitoring:type=ThreadPools`` object
+name and has following attributes:
+
++-----------------------+---------------------------------------------------------+
+| Name                  | Description                                             |
++=======================+=========================================================+
+| ``Generic``           | Thread pool statistic of the ``generic`` thread pool.   |
++-----------------------+---------------------------------------------------------+
+| ``Search``            | Thread pool statistics of the ``search`` thread pool    |
+|                       | used by read queries on user generated tables.          |
```

Comment:
uber minor but "read queries" is a pleonasm. maybe we should only use `queries`?

Reply:
would call it then read statements